French French translation of the Settings Dialog

oleg68 opened the pull request:
Fixed French translation of the Settings Dialog #934

@larspalo could you also update the swedish translation of the settings dialog?

@Mikelectric could you help us to update the german translation?

Mikelectric commented:
I can help again with the translation into German. Is the text for the new
settings included in de.po?

There are still many places in GO that have not yet been translated because
the texts were not included in de.po. Is there any prospect of changing
that?

Am Mo., 24. Jan. 2022 um 21:10 Uhr schrieb Oleg Samarin <
***@***.***>:

> Fixed French translation of the Settings Dialog #934
> <https://github.com/GrandOrgue/grandorgue/discussions/934>
>
> @larspalo <https://github.com/larspalo> could you also update the swedish
> translation of the settings dialog?
>
> @Mikelectric <https://github.com/Mikelectric> could you help us to update
> the german translation?
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/GrandOrgue/grandorgue/pull/986
> Commit Summary
>
>    - be5c16c
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/be5c16c243d9e757b015e9348fbd63e0c79b8e9e>
>    Update fr.po
>    - 612cc7d
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/612cc7d3c09f045324c134f66d20762964246811>
>    Update fr.po
>    - 304cce0
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/304cce0e1e070a1efe5e7f58c3d470ed64764b2d>
>    Update CHANGELOG.md
>    - 73177cb
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/73177cb3606cec14b77729a38e507f670185f234>
>    Merge remote-tracking branch 'Jean-Gabriel1982/patch-2' into
>    bugfix/translations
>    - 28f0d63
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/28f0d632fe68761bded789ba7afc12e28f24e63a>
>    Merge remote-tracking branch 'Jean-Gabriel1982/patch-1' into
>    bugfix/translations
>    - 5f69661
>    <https://github.com/GrandOrgue/grandorgue/pull/986/commits/5f69661c95299954f25867f0c721d45f1e7814b6>
>    Updating French translation
>    https://github.com/GrandOrgue/grandorgue/discussions/934
>
> File Changes
>
> (2 files <https://github.com/GrandOrgue/grandorgue/pull/986/files>)
>
>    - *M* CHANGELOG.md
>    <https://github.com/GrandOrgue/grandorgue/pull/986/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed>
>    (3)
>    - *M* po/fr.po
>    <https://github.com/GrandOrgue/grandorgue/pull/986/files#diff-940623802973a9e80a24d1c2e258a73d0e9c532b9feb52d2607681dca35f83c6>
>    (168)
>
> Patch Links:
>
>    - https://github.com/GrandOrgue/grandorgue/pull/986.patch
>    - https://github.com/GrandOrgue/grandorgue/pull/986.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/pull/986>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABPDN6ILTCGIM7VMDHFNHY3UXWW3DANCNFSM5MWLBL2Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

rousseldenis commented:
> I can help again with the translation into German. Is the text for the new settings included in de.po? There are still many places in GO that have not yet been translated because the texts were not included in de.po. Is there any prospect of changing that? 

@Mikelectric For sure, you can contribute. Could you do a PR with your modifications ?

Mikelectric commented:
It's still new territory for me, but I'm trying to get into Git right now.
A grandorgue fork repository is now on my GitHub page. In addition, I now
also have Git on my PC at home and cloned the repo.

Read discussion #934 and it helps a bit.
So now I'm trying to change de.po and finally make a pull request.

Maybe I can do something more in the future. I used to study computer
science, but haven't programmed in C for a long time.

Am Di., 25. Jan. 2022 um 16:13 Uhr schrieb Denis Roussel (ACSONE) <
***@***.***>:

> I can help again with the translation into German. Is the text for the new
> settings included in de.po? There are still many places in GO that have not
> yet been translated because the texts were not included in de.po. Is there
> any prospect of changing that?
>
> @Mikelectric <https://github.com/Mikelectric> For sure, you can
> contribute. Could you do a PR with your modifications ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GrandOrgue/grandorgue/pull/986#issuecomment-1021290815>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABPDN6ICSX3GVE7BTZZA54TUX24YDANCNFSM5MWLBL2Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

oleg68 commented:
> There are still many places in GO that have not yet been translated because the texts were not included in de.po. Is there any prospect of changing that?

I've inserted the new text to `de.po` in my fork: https://github.com/oleg68/GrandOrgue-official/blob/bugfix/translation-german/po/de.po

You can 
1. make a new fork from there
2. go to the bugfix/translation-german branch
3. go to the `po/po.de` file in the browser
4. Click on Edit (a pencil icon at the upper left)
5. Find all `***` patterns and replace them with a valid text in German
6. Save changes. Github will ask you for commit message
7. After commit, make a pull request from your branch not to my fork, but to the main repo

oleg68 commented:
@rousseldenis @Jean-Gabriel1982 I've added some French translation. I know you both speak French. Could you check my translation?

rousseldenis commented on po/fr.po:
```diff
@@ -3001,11 +2962,17 @@ msgstr "&Interfaces d'entrée MIDI"
+msgstr "Sortie &MIDI pour..."
+
+msgid "Matching..."
+msgstr "Correspondance"
```
Maybe '...'

oleg68 replied:
Fixed

rousseldenis commented on po/fr.po:
```diff
@@ -3001,11 +2962,17 @@ msgstr "&Interfaces d'entrée MIDI"
-msgid "Send MIDI Recorder Output Stream to "
-msgstr "Envoyer la sortie de l'enregistreur MIDI à "
+msgid "Send MIDI Recorder Output Stream to:"
+msgstr "Envoyer la sortie de l'enregistreur MIDI à:"
```
I would say 'Envoyer le flux de sortie de l'enregistreur...'

oleg68 replied:
``Envoyer le flux de sortie de l'enregistreur à:``

rousseldenis commented on po/fr.po:
```diff
@@ -3148,6 +3160,24 @@ msgstr "Taux d'échantillonage:"
+msgstr "Propriétés MIDI"
+
+msgid "Auto add new devices"
+msgstr "Ajouter les pereferiques nouveux"
```
'Ajouter automatiquement les nouveaux périphériques'

oleg68 replied:
Too long. It wont't fit.

Jean-Gabriel1982 replied:
@rousseldenis @oleg68 
 Maybe then "Ajout auto de nouveaux appareils" ?

oleg68 replied:
Your variant is still fitting
![midi-fr](https://user-images.githubusercontent.com/11586506/151412619-f28b3991-10a0-4243-bc86-7f73d88c4d88.png)

The dialog size is limited by another very long label on the Options page: ``Gérer automatiquement le cache de l'orgue lors du chargement``

![Options-fr](https://user-images.githubusercontent.com/11586506/151413012-dc1d6354-3284-4995-9c64-462a413dd8c9.png)

May be we remove ``lors du chargement``? What is your opinion?

Jean-Gabriel1982 replied:
@oleg68 I think it is better to remove "de l'orgue" instead of "lors du chargement" ... i think we can understand that it is the organ cache and not potatoes cache 🥳 

Then "Gérer automatiquement le cache lors du chargement"

As this option is in "Cache" part, maybe we can just put
"Gérer automatiquement lors du chargement"

@rousseldenis What do you think ?

Jean-Gabriel1982 replied:
> Your variant is still fitting
> 
> ![midi-fr](https://user-images.githubusercontent.com/11586506/151412619-f28b3991-10a0-4243-bc86-7f73d88c4d88.png)
> 
> 

Or less longer 
"Ajout automatique des nouveaux périphériques"
?

oleg68 replied:
Thank you, I've applied your suggestions.

As shorter as better.

rousseldenis reviewed (changes requested):
Seems ok for most.

I'm wondering if '&' characters means 'keyboard shortcuts' and if they shouldn't be in translations too?

oleg68 commented:
> I'm wondering if '&' characters means 'keyboard shortcuts' and if they shouldn't be in translations too?

It was planned so. But it doesn't work at most of places. So I don't add them to the translations.

oleg68 commented:
@Jean-Gabriel1982 @rousseldenis could you approve this PR?

oleg68 commented:
@rousseldenis @Jean-Gabriel1982 are we continuing working with the french translation or we may merge it?

larspalo commented:
I just still think it's not necessary to modify the CHANGELOG for just updating existing translations.

oleg68 commented:
> I just still think it's not necessary to modify the CHANGELOG for just updating existing translations.

If we updated the translation in the same release with the changing the Setting Dialog, then this work was a part of #934 and wouldn'r require a separate entry in CHANGELOG.

But we issued a release 3.6.0 without corresponding changes in translations, so we introduced a bug visible for users.

In the current release (3.6.1) we partially fix this bug (unfortunally my knowledge of foreign languages doesn't allow me to update translations to other languages. May be trying to update Italian also...). So I included the bugfix entry in CHANGELOG.

@larspalo if you updated the swedish translation, it would be great!

larspalo commented:
@oleg68 I'm on to it. Still, updating translations is not introducing any new features. And lack of translations is no bug - it's just working as intended.

oleg68 commented:
@larspalo is the CHANGELOG entry is the only reason why you don't approve this PR?
